Source/trigs.cpp

```diff
@@ -57,6 +57,34 @@ int L6TWarpUpList[] = { 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92,
 int L6UpList[] = { 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, -1 };
 int L6DownList[] = { 57, 58, 59, 60, 61, 62, 63, 64, -1 };
 
+const char *LvlNames[] = {
+	"Town",
+	"Dungeon 1",
+	"Dungeon 2",
+	"Dungeon 3",
+	"Dungeon 4",

```

Comment:
Cathedral

Reply:
Used Dungeon based off the text that displays for the cathedral trigger for consistency. Still change?

Reply:
> Cathedral

And how do you feel about renaming town to tristram?

Reply:
I reckon Cathedral here does read better (I think you were going that way based off the discussion in discord anyway, but adding another vote for :P)

Reply:
That trigger probably refers to all levels rather then the first one.